integration-tests/oidc-wiremock/src/main/resources/application.properties

```diff
@@ -72,6 +72,16 @@ quarkus.oidc.code-flow-user-info-github.code-grant.headers.X-Custom=XCustomHeade
 quarkus.oidc.code-flow-user-info-github.client-id=quarkus-web-app
 quarkus.oidc.code-flow-user-info-github.credentials.secret=AyM1SysPpbyDfgZld3umj1qzKObwVMkoqQ-EstJQLr_T-1qS0gZH75aKtMN3Yj0iPS4hcgUuTwjAzZr1Z9CAow
 
+quarkus.oidc.bearer-user-info-github-service.provider=github
+quarkus.oidc.bearer-user-info-github-service.verify-opaque-access-token-with-user-info=true
+quarkus.oidc.bearer-user-info-github-service.application-type=service
+quarkus.oidc.bearer-user-info-github-service.auth-server-url=${keycloak.url}/realms/quarkus/
+quarkus.oidc.bearer-user-info-github-service.authorization-path=/
+quarkus.oidc.bearer-user-info-github-service.user-info-path=protocol/openid-connect/userinfo
+quarkus.oidc.bearer-user-info-github-service.client-id=quarkus-web-app
+quarkus.oidc.bearer-user-info-github-service.credentials.secret=AyM1SysPpbyDfgZld3umj1qzKObwVMkoqQ-EstJQLr_T-1qS0gZH75aKtMN3Yj0iPS4hcgUuTwjAzZr1Z9CAow
+io.quarkus.it.keycloak.TokenPropagatingGitHubClient/mp-rest/url=http://localhost:8081/
```

Comment:
Hmm, I wonder if the test really works, oh I see, it works because you do not enforce userinfo is required property in the code

Reply:
Please see `OpaqueTokenVerificationWithUserInfoValidationTest`.

Reply:
@michalvavrik Cool, I forgot some if these properties are collapsed into a composite config like `provider=github`, thanks